src/app/tests/suites/certification/Test_TC_S_2_3.yaml

```diff
@@ -459,7 +530,214 @@ tests:
                 value: "y"
 
     - label:
-          "Step 6a: TH sends a CopyScene command to DUT with the mode field set
+          "Step 6a: TH sends a AddScene command to group G1 with the GroupID
+          field set to G1, the SceneID field set to 0x01, the TransitionTime
+          field set to 1000 (1s) and a set of extension fields appropriate to
+          AC1."
+      PICS: S.S.C00.Rsp && PICS_SDK_CI_ONLY
+      command: "AddScene"
+      groupId: G1
+      arguments:
+          values:
+              - name: "GroupID"
+                value: G1
+              - name: "SceneID"
+                value: 0x03
+              - name: "TransitionTime"
+                value: 1000
+              - name: "SceneName"
+                value: "Scene1"
+              - name: "ExtensionFieldSets"
+                value:
+                    [
+                        {
+                            ClusterID: 0x0006,
+                            AttributeValueList:
+                                [{ AttributeID: 0x0000, AttributeValue: 0x01 }],
+                        },
+                        {
+                            ClusterID: 0x0008,
+                            AttributeValueList:
+                                [{ AttributeID: 0x0000, AttributeValue: 0x64 }],
+                        },
+                    ]
+
+    - label:
+          "Step 6a: TH sends a AddScene command to group G1 with the GroupID
+          field set to G1, the SceneID field set to 0x03, the TransitionTime
+          field set to G1 and a set of extension fields appropriate to AC1."
+      verification: |
+          Execute the following command in TH to generate the pre-condition to execute this test case:
```

Comment:
what's the precondition here? This just looks like the AddScene command.

Reply:
Removed